libraries/Microsoft.Bot.Builder.Core.Extensions/IStorage.cs

```diff
@@ -48,22 +48,19 @@ public static class StorageExtensions
         /// <param name="storage"></param>
         /// <param name="keys"></param>
         /// <returns></returns>
-        public static async Task<IEnumerable<KeyValuePair<string, StoreItemT>>> Read<StoreItemT>(this IStorage storage, params string[] keys) where StoreItemT : class
+        public static async Task<IDictionary<string, StoreItemT>> Read<StoreItemT>(this IStorage storage, params string[] keys) where StoreItemT : class
         {
             var storeItems = await storage.Read(keys).ConfigureAwait(false);
-
-            return ReturnStoreItemsOfDesiredType();
-
-            IEnumerable<KeyValuePair<string, StoreItemT>> ReturnStoreItemsOfDesiredType()
+            IDictionary<string, StoreItemT> values = new Dictionary<string, StoreItemT>();
```

Comment:
The codebase seems to be `var` based, so I think we would want to stay consistent with that.

Reply:
Fixed.